src/quota_aggregator_impl.cc

```diff
@@ -258,20 +237,47 @@ ::google::protobuf::util::Status QuotaAggregatorImpl::FlushAll() {
 
 // OnCacheEntryDelete will be called behind the cache_mutex_
 // no need to consider locking at this point
+// This is called when an cached item is older than refresh_interval
```

Comment:
I am getting confused between refresh/flush interval and expiration interval. Will this also be called when expiration occurs? Or will this decide when expiration occurs via `ShouldDrop`?

Reply:
This is called on every refresh_interval.  Of course, it will be called when it is expired since expiration is much higher than refresh_interval,  the former is 1s,  the latter is 1m

Added comment to explain how it works